src/toolchain.rs

```diff
@@ -228,46 +228,75 @@ impl Toolchain {
+                                    bin.as_path(),
+                                    &self.bin_path.join(exe_file_name),
+                                )?;
+                            }
+                        }
+                    }
 
-        if let Ok(downloaded) = unpack_bins(&self.bin_path, &fuelup_bin_dir) {
-            link_to_fuelup(downloaded)?;
-
-            // Little hack here to download core and std lib upon installing `forc`
-            if download_cfg.name == component::FORC {
-                let fuelup_tmp_dir = fuelup_tmp_dir();
-                ensure_dir_exists(&fuelup_tmp_dir)?;
-                let forc_bin_path = self.bin_path.join(component::FORC);
-                let temp_project = tempfile::Builder::new().prefix("temp-project").tempdir()?;
-                let temp_project_path = temp_project.path().to_str().unwrap();
-                if Command::new(&forc_bin_path)
-                    .args(["init", "--path", temp_project_path])
-                    .stdout(std::process::Stdio::null())
-                    .status()
-                    .is_ok()
-                {
-                    info!("Fetching core forc dependencies");
-                    if Command::new(forc_bin_path)
-                        .args(["check", "--path", temp_project_path])
-                        .stdout(std::process::Stdio::null())
-                        .status()
-                        .is_err()
-                    {
-                        error!("Failed to fetch core forc dependencies");
+                    // Little hack here to download core and std lib upon installing `forc`
+                    if download_cfg.name == component::FORC {
+                        let forc_bin_path = self.bin_path.join(component::FORC);
+
+                        let fuelup_tmp_dir = fuelup_tmp_dir();
+                        ensure_dir_exists(&fuelup_tmp_dir)?;
+                        let temp_project =
+                            tempfile::Builder::new().prefix("temp-project").tempdir()?;
+                        let temp_project_path = temp_project.path().to_str().unwrap();
+                        if Command::new(&forc_bin_path)
+                            .args(["init", "--path", temp_project_path])
+                            .stdout(std::process::Stdio::null())
+                            .status()
+                            .is_ok()
+                        {
+                            info!("Fetching core forc dependencies");
+                            if Command::new(forc_bin_path)
+                                .args(["check", "--path", temp_project_path])
+                                .stdout(std::process::Stdio::null())
+                                .status()
+                                .is_err()
+                            {
+                                error!("Failed to fetch core forc dependencies");
+                            };
+                        };
```

Comment:
Unrelated to this PR, but could be nice to abstract this into a function, i.e. `fn cache_sway_std_libs`

Reply:
Yup, I was going to put that here but it made the diff ugly so I pulled it out in https://github.com/FuelLabs/fuelup/pull/372/commits/c893d718a51e70738c8dc8f3a8f9ea7df4c1c6ee :laughing: